<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAO Studio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rdo_studio.qrc">
    <normaloff>:/images/images/logo.png</normaloff>:/images/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actFileNew"/>
    <addaction name="actFileOpen"/>
    <addaction name="actFileReopen"/>
    <addaction name="actFileClose"/>
    <addaction name="separator"/>
    <addaction name="actFileSave"/>
    <addaction name="actFileSaveAs"/>
    <addaction name="actFileSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actHelpContext"/>
    <addaction name="separator"/>
    <addaction name="actHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actEditUndo"/>
    <addaction name="acteditRedo"/>
    <addaction name="separator"/>
    <addaction name="actEditCut"/>
    <addaction name="actEditCopy"/>
    <addaction name="actEditPaste"/>
    <addaction name="actEditClear"/>
    <addaction name="separator"/>
    <addaction name="actEditCopyAsRTF"/>
    <addaction name="separator"/>
    <addaction name="actEditSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actEditUpPercase"/>
    <addaction name="actEditLowPercase"/>
    <addaction name="actEditCommentSelection"/>
    <addaction name="actEditCompleteWord"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="actSearchFind"/>
    <addaction name="actSearchFindNext"/>
    <addaction name="actSearchFindPrevious"/>
    <addaction name="actSearchReplace"/>
    <addaction name="separator"/>
    <addaction name="actSearchFinInModel"/>
    <addaction name="separator"/>
    <addaction name="actSearchBookmarksAddClear"/>
    <addaction name="actSearchBookmarkNext"/>
    <addaction name="actSearchBookmarkPrevious"/>
    <addaction name="actSearchBookmarksClearAll"/>
    <addaction name="separator"/>
    <addaction name="actSearchGotoLine"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Просмотр</string>
    </property>
    <widget class="QMenu" name="menuViewToolbar">
     <property name="title">
      <string>Панели</string>
     </property>
     <addaction name="actViewToolbarFile"/>
     <addaction name="actViewToolbarEdit"/>
     <addaction name="actViewToolbarZoom"/>
     <addaction name="actViewToolbarModel"/>
    </widget>
    <addaction name="actViewToggleAllfolds"/>
    <addaction name="actViewToggleCurrentfold"/>
    <addaction name="separator"/>
    <addaction name="actViewShowSpace"/>
    <addaction name="actViewShowEndSymbol"/>
    <addaction name="separator"/>
    <addaction name="actViewZoomMax"/>
    <addaction name="actViewZoomMin"/>
    <addaction name="actViewZoomAuto"/>
    <addaction name="actViewZoomReset"/>
    <addaction name="separator"/>
    <addaction name="menuViewToolbar"/>
    <addaction name="separator"/>
    <addaction name="actViewSettings"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Вставка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Модель</string>
    </property>
    <addaction name="actModelBuild"/>
    <addaction name="actModelRun"/>
    <addaction name="actModelStop"/>
    <addaction name="separator"/>
    <addaction name="actModelRuntimeMaxSpeed"/>
    <addaction name="actModelRuntimeJump"/>
    <addaction name="actModelRuntimeSync"/>
    <addaction name="actModelRuntimePause"/>
    <addaction name="separator"/>
    <addaction name="actModelShowrateInc"/>
    <addaction name="actModelShowrateDec"/>
    <addaction name="separator"/>
    <addaction name="actModelFrameNext"/>
    <addaction name="actModelFramePrev"/>
   </widget>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>График</string>
    </property>
    <addaction name="actChartFindInChart"/>
    <addaction name="actChartAddToNewChart"/>
    <addaction name="separator"/>
    <addaction name="actChartTimeWrap"/>
    <addaction name="separator"/>
    <addaction name="actChartOptions"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Окна</string>
    </property>
    <addaction name="actWindowCascade"/>
    <addaction name="actWindowTitleHorzontal"/>
    <addaction name="actWindowArrange"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuView"/>
   <addaction name="menuInsert"/>
   <addaction name="menuModel"/>
   <addaction name="menuChart"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actFileNew">
   <property name="text">
    <string>Новая модель...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actFileOpen">
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFileReopen">
   <property name="text">
    <string>Недавние</string>
   </property>
  </action>
  <action name="actFileClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actFileSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actFileSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actFileSaveAll">
   <property name="text">
    <string>Сохранить всё</string>
   </property>
  </action>
  <action name="actFileExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actHelpContext">
   <property name="icon">
    <iconset resource="../rdo_studio.qrc">
     <normaloff>:/images/images/help_context.png</normaloff>:/images/images/help_context.png</iconset>
   </property>
   <property name="text">
    <string>Содержание</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actHelpAbout">
   <property name="icon">
    <iconset resource="../rdo_studio.qrc">
     <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="actEditUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="acteditRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actEditCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actEditCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actEditPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actEditClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actEditCopyAsRTF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Копировать как RTF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actEditSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выделить все</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actEditUpPercase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>В верхний регистр</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="actEditLowPercase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>В нижний регистр</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actEditCommentSelection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Закомментировать выделенное</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actEditCompleteWord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Завершить слово</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actSearchFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actSearchFindNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти следующий</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actSearchFindPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти предыдущий</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actSearchReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Заменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actSearchFinInModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти в модели</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actSearchBookmarksAddClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Поставить/снять закладку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actSearchBookmarkNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Перейти к след. закладке</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actSearchBookmarkPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Перейти к пред. закладке</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="actSearchBookmarksClearAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Очистить все закладки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F2</string>
   </property>
  </action>
  <action name="actSearchGotoLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Перейти на строчку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actViewToggleAllfolds">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Свернуть/развернуть все группы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actViewToggleCurrentfold">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Свернуть/развернуть текущую группу</string>
   </property>
  </action>
  <action name="actViewShowSpace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Показать пробелы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actViewShowEndSymbol">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Показать символ конца строки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actViewZoomMax">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Увеличить масштаб</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actViewZoomMin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Уменьшить масштаб</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actViewZoomAuto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Автоматический масштаб</string>
   </property>
  </action>
  <action name="actViewZoomReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Восстановить масштаб</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actViewSettings">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="actModelBuild">
   <property name="text">
    <string>Скомпилировать</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actModelRun">
   <property name="text">
    <string>Запустить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actModelStop">
   <property name="text">
    <string>Остановить</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actModelRuntimeMaxSpeed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Максимальная скорость</string>
   </property>
  </action>
  <action name="actModelRuntimeJump">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Дискретная имитация</string>
   </property>
  </action>
  <action name="actModelRuntimeSync">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Синхронная имитация</string>
   </property>
  </action>
  <action name="actModelRuntimePause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="actModelShowrateInc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Увеличить масштаб</string>
   </property>
   <property name="shortcut">
    <string>Alt++</string>
   </property>
  </action>
  <action name="actModelShowrateDec">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Уменьшить масштаб</string>
   </property>
  </action>
  <action name="actModelFrameNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>След. кадр анимации</string>
   </property>
  </action>
  <action name="actModelFramePrev">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Пред. кадр анимации</string>
   </property>
   <property name="shortcut">
    <string>Alt+-</string>
   </property>
  </action>
  <action name="actChartFindInChart">
   <property name="text">
    <string>Поиск на графике</string>
   </property>
  </action>
  <action name="actChartAddToNewChart">
   <property name="text">
    <string>Добавить на новый график</string>
   </property>
  </action>
  <action name="actChartTimeWrap">
   <property name="text">
    <string>'Схлопнуть' время</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actChartOptions">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actWindowCascade">
   <property name="text">
    <string>Расположить каскадом</string>
   </property>
  </action>
  <action name="actWindowTitleHorzontal">
   <property name="text">
    <string>Расположить заголовками</string>
   </property>
  </action>
  <action name="actWindowArrange">
   <property name="text">
    <string>Упорядочить свернутые</string>
   </property>
  </action>
  <action name="actViewToolbarFile">
   <property name="text">
    <string>Панель файлов</string>
   </property>
  </action>
  <action name="actViewToolbarEdit">
   <property name="text">
    <string>Панель редактирования</string>
   </property>
  </action>
  <action name="actViewToolbarZoom">
   <property name="text">
    <string>Панель масштаба</string>
   </property>
  </action>
  <action name="actViewToolbarModel">
   <property name="text">
    <string>Панель модели</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rdo_studio.qrc"/>
 </resources>
 <connections/>
</ui>
